content/acra/configuring-maintaining/general-configuration/acra-keys/export.md

```diff
@@ -5,10 +5,11 @@ weight: 4
 
 # export
 
-**`export`** is `acra-keys` subcommand used for exporting keys of the keystore version `v2`.
+**`export`** is `acra-keys` subcommand used for exporting keys.
+
 {{< hint info >}}
 **Note**:
-Starting from `0.95.0` `acra-keys` **`export`** supports keys exporting for keystore version `v1`.
+Before `0.95.0` `acra-keys` **`export`** supports keys exporting only for keystore version `v2`.
```

Comment:
i don't understand what do you mean here

Reply:
> Before `0.95.0` `acra-keys export` used to support only keystore version `v2`. 

Will be enough and clear I think

Reply:
```suggestion
Starting from `0.95.0` `acra-keys export` supports exporting keys from keystore `v1` and `v2`. Older versions of `acra-keys` support only `v2` keystore.
```